packages/stencil-library/src/components/icon/icon.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, getAssetPath, h, Prop } from "@stencil/core";
+
+@Component({
+    tag: 'njwds-icon'
+})
+export class Icon {
+    @Prop() icon: string
+    @Prop() size: "3" | "4" | "5" | "6" | "7" | "8" | "9" | "scale" = "3"
```

Comment:
[dust] Flagging that we should double-check with Sarah on what icon sizes she wants users to be able to choose from. I also think there's a 20x20 icon size in the Figma that isn't supported by any of the USWDS icon utility classes. 

Maybe we can create an issue for this in the repo as well?

Reply:
Thank you for the reminder! Took them out for now and will follow up to confirm sizes and create issue